python/pyspark/pipelines/tests/test_block_imperative_construct.py

```diff
@@ -0,0 +1,255 @@
+#
```

Comment:
Should be added into `dev/sparktestsupport/modules.py` so the tests are actually triggred.